projects/efg-training/resources/test/app-subnets.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_subnet" "efg_app_1" {
+    vpc_id            = "${aws_vpc.vpc.id}"
```

Comment:
I see it now. I was expecting the file to be called vpc.tf instead of networking.tf